src/app/components/table/table.ts

```diff
@@ -5243,7 +5243,7 @@ export class ColumnFilter implements AfterContentInit {
      * Enables currency input.
      * @group Props
      */
-    @Input({ transform: booleanAttribute }) currency: boolean | undefined;
+    @Input({ transform: booleanAttribute }) currency: string;
```

Comment:
I don't think this should use the booleanAttribute transform if it's no longer a boolean

Reply:
If you say so. I went with the idea of it having a few booleanAttributes for non boolean inputs. That's why I've left it as is. But I'll change it if you think it's the right call.